cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -386,6 +392,7 @@ data:
       detect_json true
       enable_monitoring true
       monitoring_type prometheus
+      split_logs_by_tag false
```

Comment:
Add the same comment for this match as well?
`# Disable splitting logs by tag, as each container produces a separate tag.`

Reply:
This section won't capture containers, these are system logs.

Reply:
Cool cool.